src/main/resources/db/changelog/init/09072021_drop_loc_id_column_from_vendor_table.yml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: 09072021_drop_loc_id_column_from_vendor_table
+      author: tesliuk.ua@gmail.com
+      changes:
```

Comment:
it's better to add preconditions before updating the existing data model. So, before deleting the loc_id column we should add a precondition section to the liquiscript to be sure that the loc_id exists in the table before the deletion.
It looks like this (not formatted):
```
author: tesliuk.ua@gmail.com
preConditions:
        - onFail: MARK_RAN
        - not:
            - columnExists:
                columnName: loc_id
                tableName: vendor
changes: ...
```
https://docs.liquibase.com/concepts/advanced/preconditions.html

Reply:
"not" was not correct in this case. I've  fixed it to "and" in "preConditions"

Reply:
Yes, I just copy-pasted the code snippet with "not" from somewhere else. But you should definitely check the code snippets in the comment of course.